### PR TITLE
Don't show client info, when the app is disabled

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by blizzz — APPROVED** [review]
Better be moved as personal settings page to the app in general, but …  OK for now. Code looks good.

File: settings/templates/personal.php [hunk 1]
```diff
@@ -301,6 +301,7 @@
 </form>
 
 
+<?php if(OC_APP::isEnabled('firstrunwizard')) {?>
```
Comment (schiessle) [inline settings/templates/personal.php]: I would prefer to pass it as a template variable instead of using (legacy) functions in the template

**Review by schiessle — CHANGES_REQUESTED** [review]

File: settings/templates/personal.php (hunk 1 above)
Comment (nickvergessen) [inline settings/templates/personal.php]: I just moved this.
This is stable12 want to keep it simple.
For 13 i will move it to the app directly

**Review by schiessle — APPROVED** [review]

File: settings/templates/personal.php (hunk 1 above)
Comment (nickvergessen) [inline settings/templates/personal.php]: Added to my todo list

**Pull request merged**